scripts/apps/authoring/authoring/directives/AuthoringDirective.js

```diff
@@ -335,9 +335,11 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
             }
 
             function validate(orig, item) {
+            	var updated = {};
```

Comment:
Is this to prevent cases when `orig` is null? If so, we can assign a default value by changing the function signature as follows:
```js
function validate(orig = {}, item) {
```
This is an ES6 feature we can use.

Either way, the indentation is a bit off here, I think you are missing a space.